internals/notifier_test.py

```diff
@@ -1191,7 +1191,7 @@ def test_make_activated_email(self):
       handler = notifier.OTActivatedHandler()
       stage_dict = converters.stage_to_json_dict(self.ot_stage)
       email_task = handler.build_email(stage_dict, self.contacts)
-      TESTDATA.make_golden(email_task['html'], 'test_make_activated_email.html')
+      # TESTDATA.make_golden(email_task['html'], 'test_make_activated_email.html')
```

Comment:
Unfortunately this was missed on my part... Luckily, these emails have not started going out yet.